OneBlink.SDK/PdfClient.cs

```diff
@@ -17,25 +18,16 @@ public PdfClient(string accessKey, string secretKey, TenantName tenantName = Ten
                 tenant: new Tenant(tenantName)
             );
         }
-        public async Task<Stream> GetSubmissionPdf(long formId, string submissionId, bool? isDraft = null, bool? includeSubmissionIdInPdf = null, List<Guid> excludedElementIds = null, bool? usePagesAsBreaks = null)
+        public async Task<Stream> GetSubmissionPdf(long formId, string submissionId, bool? isDraft = null, bool? includeSubmissionIdInPdf = null, List<Guid> excludedElementIds = null, bool? usePagesAsBreaks = null, bool? includePaymentInPdf = null)
         {
-            string queryString = string.Empty;
-            if (isDraft.HasValue)
-            {
-                queryString += "isDraft=" + isDraft.Value.ToString();
-            }
-            if (includeSubmissionIdInPdf.HasValue)
-            {
-                if (queryString != string.Empty)
-                {
-                    queryString += "&";
-                }
-                queryString += "includeSubmissionIdInPdf=" + includeSubmissionIdInPdf.Value.ToString();
-            }
-            GetSubmissionPdfRequest body = new GetSubmissionPdfRequest() { excludedElementIds = excludedElementIds, usePagesAsBreaks = usePagesAsBreaks };
+            var query = HttpUtility.ParseQueryString(string.Empty);
+            OneBlinkHttpClient.AddItemToQuery(query, nameof(isDraft), isDraft);
+            OneBlinkHttpClient.AddItemToQuery(query, nameof(includeSubmissionIdInPdf), includeSubmissionIdInPdf);
+            OneBlinkHttpClient.AddItemToQuery(query, nameof(includePaymentInPdf), includePaymentInPdf);

```

Comment:
Can these all be moved to the body of the request please.

Reply:
Has the API been changed?

Reply:
it will be